code/src/styles.js

```diff
@@ -0,0 +1,194 @@
+import styled from 'styled-components'
+import { createGlobalStyle } from 'styled-components'
+
+export const devices = {
+    mobile: "(min-width: 375px)",
+    tablet: "(min-width: 768px)",
+    desktop: "(min-width: 1025px)",
+}
+
+export const GlobalStyle = createGlobalStyle`  
```

Comment:
Really cool that you used the GlobalStyle, it was something I was hoping to use but didn't have time to. So I learned a lot from your code, thank you!